data/ratings.csv

```diff
@@ -89,4 +89,3 @@ userId,categoryId,rating,tags
 1,3,10,stars|galaxy|spaceship|magic
 1,3,9,phoenix|magic|castle|wizard
 1,3,8,unicorn|fire|knight|forest
-1,1,10,waterfall|forest|trees|rainbow
```

Comment:
Why?

Reply:
Cause last comments was why was it in commit and it was in there by mistake (not deleted after testing)